Source/WebCore/html/canvas/CanvasRenderingContext2DBase.cpp

```diff
@@ -2350,9 +2350,11 @@ void CanvasRenderingContext2DBase::didDraw(std::optional<FloatRect> rect, Option
     if (m_dirtyRect.contains(dirtyRect))
         canvasBase().didDraw(std::nullopt, shouldApplyPostProcessing);
     else {
+#if !PLATFORM(GTK) && !PLATFORM(WPE)
```

Comment:
Is this because of Skia? or CoordinatedGraphics? or linux? I think we should use the right condition here instead of GTK/WPE